server/utils/next_url.test.ts

```diff
@@ -18,14 +18,16 @@ import {
   validateNextUrl,
   INVALID_NEXT_URL_PARAMETER_MESSAGE,
 } from './next_url';
+import { httpServerMock } from '../../../../src/core/server/mocks';
```

Comment:
@SuZhou-Joe do you think we should add tests here for workspace related paths? Or is that overkill since it is tested by util function in core already?

Reply:
Yeah, it has been covered by util function in core already.

Reply:
But I do think we need a integration test to cover the case when user logout in a workspace url,  raised an [issue](https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/1519) for tracking.

Reply:
Cool if functional test is added for 2.17, I don't have many concerns on this front.